lib/components/radio/radio.style.dart

```diff
@@ -16,14 +16,13 @@ class RadioStyles extends StyleRecipe<RadioStyles> {
   final Style label;
   final Style row;
 
-  factory RadioStyles.build(
-      [RadioStyles? other, List<Variant> variants = const []]) {
+  factory RadioStyles.baseForm() {
```

Comment:
Lets use `base` instead of `baseForm`

Reply:
I guessed `base` was a keyword, but when I tested it just now, it turned out not to be

Reply:
Actually `base` is a keyword https://dart.dev/language/class-modifiers#base